index.js

```diff
@@ -112,10 +140,10 @@ function sign (keys, msg) {
 function verify (keys, sig, msg) {
   if(isObject(sig))
     throw new Error('signature should be base64 string, did you mean verifyObj(public, signed_obj)')
-  return curves[getCurve(keys)].verify(
+  return feedTypes[getFeedType(keys)].verify(
     u.toBuffer(keys.public || keys),
     u.toBuffer(sig),
-    isBuffer(msg) ? msg : new Buffer(msg)
+    isBuffer(msg) ? msg : Buffer.from(msg)
```

Comment:
`<non-expert>` I think it might be a good idea to explicitly declare the encoding with `Buffer.from`

i.e.

```js
Buffer.from(msg, 'base64')
Buffer.from(msg, 'utf8') // I think this is the default
```